specification/iothub/resource-manager/Microsoft.Devices/preview/2022-04-30-preview/iothub.json

```diff
@@ -0,0 +1,4637 @@
+      "get": {
+        "tags": [
+          "GET"
+        ],
+        "summary": "Get the non-security related metadata of an IoT hub",
+        "description": "Get the non-security related metadata of an IoT hub.",
+        "operationId": "IotHubResource_Get",
+        "x-ms-examples": {
+          "IotHubResource_Get": {
+            "$ref": "./examples/iothub_get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The body contains all the non-security properties of the IoT hub. Security-related properties are set to null.",
+            "schema": {
+              "$ref": "#/definitions/IotHubDescription"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "put": {
+        "tags": [
+          "PUT"
+        ],
+        "summary": "Create or update the metadata of an IoT hub.",
+        "description": "Create or update the metadata of an Iot hub. The usual pattern to modify a property is to retrieve the IoT hub metadata and security metadata, and then combine them with the modified values in a new body to update the IoT hub.",
+        "operationId": "IotHubResource_CreateOrUpdate",
+        "x-ms-examples": {
+          "IotHubResource_CreateOrUpdate": {
+            "$ref": "./examples/iothub_createOrUpdate.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/resourceName"
+          },
+          {
+            "name": "iotHubDescription",
+            "in": "body",
+            "description": "The IoT hub metadata and security metadata.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/IotHubDescription"
+            }
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "required": false,
+            "type": "string",
+            "description": "ETag of the IoT Hub. Do not specify for creating a brand new IoT Hub. Required to update an existing IoT Hub."
+          }
+        ],
+        "responses": {
+          "201": {
+            "headers": {
```

Comment:
This should be the definition of header instead of the value.
Pls refer to https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md#headers-object.